Add Storybook Story to Label Component

natashaU opened the pull request:
- This PR adds a [storybook story](https://github.com/jonspalmer/view_component_storybook) to the Label Component:

<img width="820" alt="Screen Shot 2020-09-17 at 8 13 05 PM" src="https://user-images.githubusercontent.com/18093541/93540624-46305b80-f922-11ea-9066-a41ad082cbab.png">

- I also added `bundle install;` to the procfile. Not sure we want to include that in this PR ❓ ❓ , but it seemed to solve gem dependency issues whenever I switched branches and needed to start the server. 

I had the same problems as described in this Issue in regards to the `type` discrepancy: https://github.com/primer/view_components/issues/82

cc/ @joelhawksley @aellispierce @jonspalmer

manuelpuyol commented:
somehow coverage was affected 🤔

joelhawksley commented on Procfile:
```diff
@@ -1,3 +1,3 @@
-stories: cd demo; bin/rails view_component_storybook:write_stories_json
-rails: cd demo; STORYBOOK=true bin/rails s -p 4000
+stories: cd demo; bundle install; bin/rails view_component_storybook:write_stories_json
```
@manuelpuyol any idea how to avoid needing this `bundle install`?

manuelpuyol replied:
I'm not sure, I haven't run into the issues described tbh 🤔

natashaU commented:
@manuelpuyol I removed the stories helper array as per your recent PR update: https://github.com/primer/view_components/pull/91
@joelhawksley I removed `.to_sym` as per your recent PR with the gem upgrade: https://github.com/primer/view_components/pull/89

Everything looks good ~~

manuelpuyol commented on stories/primer/label_component_stories.rb:
```diff
@@ -0,0 +1,17 @@
+  story(:label) do
+    controls do
+      title "this is a label"
+      select(:scheme, (Primer::LabelComponent::SCHEME_MAPPINGS.keys), :blue)
+      select(:variant, (Primer::LabelComponent::VARIANT_MAPPINGS.keys), :large)
```
```suggestion
      select(:scheme, Primer::LabelComponent::SCHEME_MAPPINGS.keys, :blue)
      select(:variant, Primer::LabelComponent::VARIANT_MAPPINGS.keys, :large)
```
